### PR TITLE
🖍 Fixes display of small percentages in binary polls.

### PR DESCRIPTION
**Pull request by processprocess**
### [Demo](https://poll-animation.web.app/examples/amp-story/interactives.html)

Small percentages are now visible.
[screenshot](https://drive.google.com/file/d/1Z4GUjbleGixGs_J8xE8ApmbJXqPrDKYL/view?usp=sharing)

Before, small percentages were clipped.
[screenshot](https://drive.google.com/file/d/1-l3Ugk8pIoNyjvTMFmyI3jmLuBAVU1ml/view?usp=sharing)

If a poll is 0% it is not visible.
[screenshot](https://drive.google.com/file/d/18EkZyCoENepyZAQ41OWnz8PfPkYg2ZgS/view?usp=sharing)

### CODE REVIEW

**Comment by processprocess** [comment]
@mszylkowski

File: extensions/amp-story/1.0/amp-story-interactive-poll.css [hunk 1]
```diff
@@ -42,7 +42,7 @@
   flex-direction: column !important;
   flex-grow: 50 !important;
   flex: 100 !important;
-  overflow: hidden !important;
+  min-width: 0;
```
Comment (Enriqe) [inline extensions/amp-story/1.0/amp-story-interactive-poll.css:45]:
```suggestion
  min-width: 0 !important;
```

File: extensions/amp-story/1.0/amp-story-interactive-poll.js [hunk 2]
```diff
@@ -202,6 +202,19 @@ export class AmpStoryInteractivePoll extends AmpStoryInteractive {
...
       ).textContent = `${percentage}%`;
       currOption.setAttribute('style', `flex-grow: ${percentage} !important`);
+
+      if (percentage < 20) {
+        const textContainer = currOption.querySelector(
+          '.i-amphtml-story-interactive-option-text-container'
+        );
+        textContainer.setAttribute(
+          'style',
+          `transform: translateX(${index === 0 ? 27 : -27}%) !important`
```
Comment (Enriqe) [inline extensions/amp-story/1.0/amp-story-interactive-poll.js:212]: Can we make this `27` a top file constant and call it `MIN_HORIZONTAL_PADDING_{LEFT|RIGHT}` or something like that?

Comment (processprocess) [reply extensions/amp-story/1.0/amp-story-interactive-poll.js]: Done.

**Comment by processprocess** [comment]
#29060 is an alternate approach that uses transforms only.

File: extensions/amp-story/1.0/amp-story-interactive-poll.js [hunk 3]
```diff
@@ -202,6 +205,21 @@ export class AmpStoryInteractivePoll extends AmpStoryInteractive {
         '.i-amphtml-story-interactive-option-percentage-text'
       ).textContent = `${percentage}%`;
       currOption.setAttribute('style', `flex-grow: ${percentage} !important`);
+
+      if (percentage < 20) {
+        const textContainer = currOption.querySelector(
+          '.i-amphtml-story-interactive-option-text-container'
+        );
+        textContainer.setAttribute(
```
Comment (gmajoulet) [inline extensions/amp-story/1.0/amp-story-interactive-poll.js:213]: Missed this during hte previous review sorry, please use the setStyles() helper so you don't override already present styles

Comment (mszylkowski) [reply extensions/amp-story/1.0/amp-story-interactive-poll.js:213]: Just commented this to the PR that we will actually use 😂

**Pull request closed without merging**